### DISCUSSION TITLE
Installation geht nicht auf 7583

### DISCUSSION THREAD

**Discussion by miodzicho** [Freetz-NG/freetz-ng#759] [Allgemein]
![image](https://user-images.githubusercontent.com/1526145/224145106-ee2f565b-69d0-4a43-a65f-f32399aa2b38.png)
Ich kann ohne Ende warten, kommt nicht. Keine ping Antwort, keine LED lichter. Nach hard reboot kommt nur ftp, keine Web Oberfläche.

**Comment by fda77** [comment]
Wenn während "wait" nichts erkannt wird könntest du mal den stromstecker der box rausziehen und wieder reinstecken

**Comment by miodzicho** [comment]
Hat auch nicht geklappt...habe ich auch probiert.

![image](https://user-images.githubusercontent.com/1526145/224632808-c8b67a33-ef35-4daa-8624-99562c7801ba.png)
